From f47a2ecce3ade2ee131b345956ebe1dc0705885a Mon Sep 17 00:00:00 2001
From: alfsamsung <alfsamsung@hotmail.com>
Date: Sat, 3 Jan 2015 01:40:20 +0100
Subject: [PATCH] STOP_LOG_SPAM

---
 core/jni/android_os_Debug.cpp | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/core/jni/android_os_Debug.cpp b/core/jni/android_os_Debug.cpp
index 7b3528b..f3b5635 100644
--- a/core/jni/android_os_Debug.cpp
+++ b/core/jni/android_os_Debug.cpp
@@ -955,6 +955,7 @@ static JNINativeMethod gMethods[] = {
 
 int register_android_os_Debug(JNIEnv *env)
 {
+#if !defined(STOP_LOG_SPAM)
     int err = memtrack_init();
     if (err != 0) {
         memtrackLoaded = false;
@@ -962,6 +963,7 @@ int register_android_os_Debug(JNIEnv *env)
     } else {
         memtrackLoaded = true;
     }
+#endif
 
     jclass clazz = env->FindClass("android/os/Debug$MemoryInfo");
 
-- 
1.9.1

